src/makeRobot.js

```diff
@@ -38,7 +38,57 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0, y: 0,
+    },
+
+    goForward(distance = 1) {
+      this.coords.y = this.coords.y + Math.max(0, distance);
+
+      return this;
```

Comment:
No need to lokking max value, just add check that value more than 0
Fix all methods
```suggestion
      if (distance > 0) {
         this.coords.y += distance;
      }

      return this;
```